### ISSUE TITLE
Defining New Datasets in Cityscapes Format

### ISSUE THREAD

**Issue by bsafacicek** [uber-research/UPSNet#133]
Hi, 

I am trying to write my own dataloader for using your codebase on KITTI semantic dataset (http://www.cvlibs.net/datasets/kitti/eval_semantics.php). For this, the easiest way I could think of was to convert KITTI labels into Cityscapes format and use your code for Cityscapes training. 

But, the codebase is expecting the following 3 files:
instancesonly_gtFine_train.json, instancesonly_gtFine_val.json, cityscapes_fine_val.json.

1) Can you provide the preprocessing code you use to get these JSON files?

2) A related question: While training on default Cityscapes, I realized that numbers of training and validation samples are 2965 and 492 instead of 2975, 500. Why 10 and 8 samples from training and validation sets are missing?

Thanks.
